test/integration/scheduler/scheduler_test.go

```diff
@@ -495,35 +276,22 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			framework.ExpectResourceFlavorToBeDeleted(ctx, k8sClient, onDemandFlavor, true)
 		})
 
-		ginkgo.It("Should schedule jobs from the selected namespaces after label update", func() {
-			ginkgo.By("checking jobs don't start at first")
-			job1 := testing.MakeJob("job1", ns.Name).Queue(queue.Name).Request(corev1.ResourceCPU, "1").Obj()
-			gomega.Expect(k8sClient.Create(ctx, job1)).Should(gomega.Succeed())
-			createdJob1 := &batchv1.Job{}
-			gomega.Eventually(func() *bool {
-				gomega.Expect(k8sClient.Get(ctx, types.NamespacedName{Name: job1.Name, Namespace: job1.Namespace}, createdJob1)).Should(gomega.Succeed())
-				return createdJob1.Spec.Suspend
-			}, framework.Timeout, framework.Interval).Should(gomega.Equal(pointer.Bool(true)), "Job should be suspended")
-
-			job2 := testing.MakeJob("job2", nsFoo.Name).Queue(queueFoo.Name).Request(corev1.ResourceCPU, "1").Obj()
-			gomega.Expect(k8sClient.Create(ctx, job2)).Should(gomega.Succeed())
-			createdJob2 := &batchv1.Job{}
-			gomega.Eventually(func() *bool {
-				gomega.Expect(k8sClient.Get(ctx, types.NamespacedName{Name: job2.Name, Namespace: job2.Namespace}, createdJob2)).Should(gomega.Succeed())
-				return createdJob2.Spec.Suspend
-			}, framework.Timeout, framework.Interval).Should(gomega.Equal(pointer.Bool(true)), "Job should be suspended")
-
-			framework.ExpectPendingWorkloadsMetric(cq, 2)
+		ginkgo.It("Should admit workloads from the selected namespaces", func() {
+			ginkgo.By("checking a workload does not get admitted at first")
```

Comment:
The test had two workloads. Can you do the same?

Reply:
Added. Refactored the tests with two workloads.